docs/website/01_framework/controlRoutines.md

```diff
@@ -75,10 +79,10 @@ void evaluateSubsidiaryFunction(SubsidiaryArgs& args);
 void initialPrediction(NonLinearOperator& nonLinearOperator, SubsidiaryArgs& args);
```

Comment:
can you add for `void evaluateSubsidiaryFunction(SubsidiaryArgs& args);`

a const at the end. `void evaluateSubsidiaryFunction(SubsidiaryArgs& args) const;`, since this belows to the functions signature

Reply:
done